help/de/Help_ch_GUIMenus.xml

```diff
@@ -3167,17 +3185,44 @@
       </para>
 
       <tip>
-        <para>Die Spalten, die für jedes Konto in der Kontenhierarchie angezeigt werden, können an Ihre
-          Bedürfnisse angepasst werden. Klicken Sie auf das Pfeilsymbol ganz rechts im Fenster
-          Kontenhierarchie um eine Dropdown-Liste mit Spaltenoptionen anzuzeigen. Klicken Sie auf
-          eine Spaltenbeschreibung, um sie zum Fenster "Kontenhierarchie" hinzuzufügen. Klicken Sie
-          erneut auf die Spaltenbeschreibung, um sie auszublenden.
+        <para>Die Spalten, die für jedes Konto in der Kontenübersicht angezeigt werden, können an Ihre
+          Bedürfnisse angepasst werden.
+        </para>
+
+        <figure pgwide="1">
+          <title>Der Spaltenselektor</title><screenshot>
+            <mediaobject>
+              <imageobject>
+                <imagedata fileref="figures/column-selector-0.png" format="PNG"
+                   srccredit="Frank H. Ellenberger" />
+              </imageobject>
+            </mediaobject>
+          </screenshot>
+        </figure>
+
+        <para>Klicken Sie auf das Pfeilsymbol ganz rechts im Fenster Kontenübersicht (Spaltenselektor) um eine
+          Dropdown-Liste mit Spaltennamen anzuzeigen.
+        </para>
+
+        <figure pgwide="1">
+          <title>Die Spaltenauswahl im Spaltenselektor</title><screenshot>
+            <mediaobject>
+              <imageobject>
+                <imagedata fileref="figures/column-selector-1.png" format="PNG"
+                   srccredit="Frank H. Ellenberger" />
+              </imageobject>
+            </mediaobject>
+          </screenshot>
+        </figure>
+
+        <para>Klicken Sie auf eine Spaltenbeschreibung, um sie zum Fenster <guilabel>Kontenübersicht</guilabel>
+          hinzuzufügen. Klicken Sie erneut auf die Spaltenbeschreibung, um sie auszublenden.
         </para>
```

Comment:
Ein Verweis wäre noch schön, sinngemäß:
Die meisten Spalten sind in XRef Edit->Account beschrieben…
Dort vielleicht noch Verweise auf Beschreibungen weiterer Spalten einfügen.